great_expectations/expectations/core/expect_column_values_to_not_be_null.py

```diff
@@ -90,7 +90,7 @@ class ExpectColumnValuesToNotBeNull(ColumnMapExpectation):
     }
 
     map_metric: ClassVar[str] = "column_values.nonnull"
-    args_keys: ClassVar[tuple[str, ...]] = ("column",)
+    args_keys: ClassVar[Tuple[str, ...]] = ("column",)
```

Comment:
All these typing changes are to appease Pydantic

Reply:
So does pydantic not support `list`, `dict`, `set`, etc?

Reply:
It does but `tuple`s are immutable and therefore better 😄

Reply:
Sorry, I was referring to `tuple` vs `Tuple` and `list` vs `List`

Reply:
Yeah, sorry this is the same reason we have to use `Union[str, int]` instead of `str | int`.
https://github.com/great-expectations/great_expectations/pull/8965#discussion_r1397834916